playground/backend/internal/tasks/task.go

```diff
@@ -34,11 +34,11 @@ func New(ctx context.Context) *ScheduledTask {
 	return &ScheduledTask{ctx: ctx, taskScheduler: chrono.NewDefaultTaskScheduler()}
 }
 
-func (st *ScheduledTask) StartRemovingExtraSnippets(cron string, dayDiff int32, db db.Database) error {
+func (st *ScheduledTask) StartRemovingExtraSnippets(cron string, externalFunction external_functions.ExternalFunctions) error {
 	task, err := st.taskScheduler.ScheduleWithCron(func(ctx context.Context) {
 		logger.Info("ScheduledTask: StartRemovingExtraSnippets() is running...\n")
 		startDate := time.Now()
-		if err := db.DeleteUnusedSnippets(ctx, dayDiff); err != nil {
+		if err := externalFunction.CleanupSnippets(ctx); err != nil {
```

Comment:
Any reason to call the externalFunction directly here, but use datastore_db as the mechanism for doing it elsewhere?

Reply:
I think that sometimes using datastore_db and sometimes using the externalFunctions is a little confusing, I'd prefer we standardize on one (my vote would be to standardize on calling externalFunctions directly since they're not inherently tied to always invoking the datastore_db.

Reply:
No particular reason for a direct call here, it's just there is not need for this function to continue to work without cloudfunctions as it's not involved in the integration testing of ToB

Reply:
I think I would like for this line to be made clearer in the future, but I'm ok moving forward with it as is for now.